src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid implements Figure {
+    private String color;
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        this.color = color;
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * (base1 + base2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + getArea() + " sq. units, base1: " + base1 + " units, base2: " + base2 + " units, height: " + height + " units, color: " + color);
```

Comment:
Avoid using System.out.println in the draw method for printing the object's details. Consider using a method that returns the String representation of the object, and then print it from elsewhere in your application. This will improve the separation of concerns and make your code more testable.